### PR TITLE
Brief template change button layout

### PR DESCRIPTION
**Pull request by Blackfry**

### CODE REVIEW

File: app/templates/brief.html [hunk 1]
```diff
@@ -123,37 +123,69 @@
...
     {% if is_restricted_brief %}<p>Only invited sellers can apply for 'Open to one' or 'Open to selected' briefs.</p>
     {% endif %}
-    <p>
 
 
-      {% with
+     {% if profile_application_status == 'approved' %}
+      <h3>Interested in this opportunity?</h3>
+      <div>
+        As you're not active on the Marketplace you will need to pass assessment before you can respond to this
```
Comment (matt-sm) [inline app/templates/brief.html]: Doesn't this mean they are active?

File: app/main/views/marketplace.py [hunk 2]
```diff
@@ -269,6 +269,13 @@ def get_brief_by_id(framework_slug, brief_id):
     application_url = "/sellers/opportunities/{}/responses/create".format(brief['id'])
     add_case_study_url = None
 
+    # pass application status to brief
+    profile_application_status = None
+    if current_user.is_authenticated and current_user.role == 'supplier':
+        profile_application_id = current_user.application_id
```
Comment (matt-sm) [inline app/main/views/marketplace.py]: gotta think about the edit case: you can have a status='submitted' but type ='edit'

File: app/templates/brief.html (hunk 1 above)
Comment (Blackfry) [inline app/templates/brief.html]: good catch

File: app/main/views/marketplace.py (hunk 2 above)
Comment (Blackfry) [inline app/main/views/marketplace.py]: I don't think type status changes the flow on brief application, you are either 'approved' and go to the application or you are submitted and get a 'assessment is underway' msg.

Comment (matt-sm) [inline app/main/views/marketplace.py]: for a supplier I would not rely on user.application_id, I would do back to application table and query on supplier_code

**Pull request merged**